lib/python/src/bailo/helper/model.py

```diff
@@ -456,6 +482,7 @@ def from_mlflow(self, tracking_uri: str, experiment_id: str):
 
             if len(mlflow.artifacts.list_artifacts(artifact_uri=artifact_uri)):
                 mlflow_dir = os.path.join(self.temp_dir, f"mlflow_{run_id}")
+                print(artifact_uri)
```

Comment:
Is this a print statement function from testing?

Reply:
Removed